docs/src/main/sphinx/connector/kafka.rst

```diff
@@ -61,30 +61,17 @@ The following configuration properties are available:
 ========================================================== ==============================================================================
 Property Name                                              Description
 ========================================================== ==============================================================================
-``kafka.table-names``                                      List of all tables provided by the catalog
 ``kafka.default-schema``                                   Default schema name for tables
 ``kafka.nodes``                                            List of nodes in the Kafka cluster
 ``kafka.buffer-size``                                      Kafka read buffer size
-``kafka.table-description-dir``                            Directory containing topic description files
 ``kafka.hide-internal-columns``                            Controls whether internal columns are part of the table schema or not
 ``kafka.messages-per-split``                               Number of messages that are processed by each Trino split, defaults to 100000
 ``kafka.timestamp-upper-bound-force-push-down-enabled``    Controls if upper bound timestamp push down is enabled for topics using ``CreateTime`` mode
 ========================================================== ==============================================================================
```

Comment:
Can you include the kafka.table-description-supplier config?
```
``kafka.table-description-supplier``                       The table description supplier to use
```

Reply:
No .. it is documented separately and the sentence after the table links to all that config